src/ray/raylet/worker_pool.h

```diff
@@ -38,6 +39,12 @@ namespace ray {
 
 namespace raylet {
 
+#ifdef _WIN32
```

Comment:
This shouldn't be needed since it is already in src/ray/raylet/worker.h. Just make sure it is in the right namespace there.